tendrl/commons/message.py

```diff
@@ -153,7 +153,7 @@ def serialize_message(obj):
         serial = obj.isoformat()
         return serial
     elif isinstance(obj, Exception):
-        return yaml.safe_dump(obj)
+        return yaml.safe_dump(obj.message)
```

Comment:
The problem here is that ruamel.yaml or pyyaml wont understand how to represent your custom Exception object. Check below two fixes and make similar fixes here, youll need to add a way to represent your Exception object and let the yaml.dump know about that

- https://github.com/openshift/openshift-ansible/pull/3156/files
- https://github.com/ansible/ansible/blob/devel/lib/ansible/parsing/yaml/dumper.py#L31